src/style.rs

```diff
@@ -398,7 +410,7 @@ impl<'a> WideElement<'a> {
                 buf.write_fmt(format_args!(
                     "{}",
                     PaddedStringDisplay {
-                        str: &style.message,
+                        str: &style.message.replace('\t', &" ".repeat(style.tab_width)),
```

Comment:
We can also pre-replace in messages.

Reply:
In this case we'd also need to store the original message and re-expand it if tab width changes. Are you OK with that complexity?

Reply:
(same with prefix)